app/models/MemberStateModel.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.{Format, Json}
+
+case class MemberStateModel(countryCode: String, country: String)
```

Comment:
Might be better named as `CountryModel` as really that is what it represents. It just so happens that we're getting a sub-set of MemberStates.

If this was renamed then it could be re-used as there will be other pages that retrieve all countries from Reference Data in future.